conans/client/tools/scm.py

```diff
@@ -81,9 +81,13 @@ def _remove_credentials_url(cls, url):
 class Git(SCMBase):
     cmd_command = "git"
 
+    @property
     def _configure_ssl_verify(self):
-        # TODO: This should be a context manager
-        return self.run("config http.sslVerify %s" % ("true" if self._verify_ssl else "false"))
+        return "-c http.sslVerify=%s " % ("true" if self._verify_ssl else "false")
+
+    def run(self, command):
+        command = self._configure_ssl_verify + command
```

Comment:
Do all git commads have the possibility to use ``-c http.sslVerify=``? I am not sure. Take into account that this ``run()`` method will be used to run different git commands as a tool, not only the commands performed for SCM as we have this class exposed in ``conans.tools``

Reply:
yes, it's common syntax which applies to all commands:
```
       git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p|--paginate|-P|--no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>]
           <command> [<args>]
```
```
       -c <name>=<value>
           Pass a configuration parameter to the command. The value given will override values from configuration files. The <name> is expected in the same format as listed by git config (subkeys separated by
           dots).
```

Reply:
ok ok, didn't see that! Thanks 👍

Reply:
Have you checked just in case if that option is supported in older (something reasonable) git clients?

Reply:
it's at least from version 1.7.2: https://github.com/git/git/commit/8b1fa778676

Reply:
Ok, let's see if no-one complains, I wouldn't bet...

Reply:
Yes, it seems reasonable, from 2010, but lets see...
In any case, I think this seems better, because there are other commands that will need it too (like checkout with submodules), and it is also good that it won't change any configuration in disk, everything will be command line